tags/README.md

```diff
@@ -1,86 +1,216 @@
+
+## Operating Model
+
+Important: Each TAG is supported by a named member of the CNCF executive staff who is accountable for liaison with the CNCF Executive Director, plus communication and performance of the TAG, with quarterly and annual reporting to Governing Board & TOC.
+
+As a starting point let’s be inspired by CNCF OSS Projects and by K8s SIGs.  That means minimal viable governance and community-based organisation.
+
+### TAG Formation, Leadership and Membership Composition
+
+1. TAGs are formed by the TOC.  Initial TAGs are listed in [proposed TAGs](/tags/proposed.md), and will be adapted over time as required.  If members of the community believe that additional TAGs are desired, they should propose these to the TOC, with clear justification, and ideally volunteers to lead the TAG. The TOC wishes to have the smallest viable number of TAGs, and for all of them to be highly effective (as opposed to a "TAG sprawl" with large numbers of relatively ineffective TAGS).
+2. TAG has three co-chairs, who are TOC Contributors, recognized as experts in that area, and for their ability to co-lead the TAG to produce the required unbiased outputs.
+3. TAG has one TOC liaison who is a voting member of the TOC acting as an additional non-executive chair on occasions when TOC input is deemed necessary by the TOC or the TAG chairs.
+4. TAG has multiple tech leads who are recognized as (1) experts in the TAG area, (2) leaders of projects in the TAG’s area (3) demonstrating the ability to provide the balanced technical leadership required to produce the required unbiased outputs of the TAG. The reason for having separate chair and tech lead roles is to allow responsibility for primarily administrative functions to be separated from deep technical functions and associated time commitments and skill sets.  Where appropriate, an individual may perform both roles as shown in [TAG member roles](/tags/cncf-tags.md#TAG-Member-Roles).
+5. Participant diversity is strongly encouraged within TAGs, and TAG chairs are expected to actively encourage a diverse range of community members to take up named roles (see point 7). 
+6. A variety of perspectives is strongly encouraged within TAGs.  To this end, a supermajority (⅔ or more) of chairs or a supermajority of tech leads from a single group of related companies, market segment, etc will be actively discouraged by the TOC.
+7. TAG members are self-declared, so that some TAG work is done by volunteers from the TOC Contributors and community.  To recognise members who make sustained  and valuable contributions to a TAG over time, TAG-defined and assigned roles may be created (e.g. scribe, training or documentation coordinator etc).  TAG’s should document what these roles and responsibilities are, and who performs them, and have them approved by TAG leads. TAGs roles should have active mentoring and shadowing programs to encourage sustainability of the TAG.
```

Comment:
```suggestion
7. TAG members are self-declared, so that some TAG work is done by volunteers from the community.  To recognise members who make sustained  and valuable contributions to a TAG over time, TAG-defined and assigned roles may be created (e.g. scribe, training or documentation coordinator etc).  TAG’s should document what these roles and responsibilities are, and who performs them, and have them approved by TAG leads. TAGs roles should have active mentoring and shadowing programs to encourage sustainability of the TAG.
```